jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -6,10 +6,28 @@ REPO_ORG="${1:-metal3-io}"
 REPO_NAME="${2:-metal3-dev-env}"
 REPO_BRANCH="${3:-master}"
 UPDATED_REPO="${4:-https://github.com/${REPO_ORG}/${REPO_NAME}.git}"
-export CAPI_VERSION="${5:-v1alpha3}"
-export IMAGE_OS="${6:-Ubuntu}"
-export DEFAULT_HOSTS_MEMORY="${7:-4096}"
-DISTRIBUTION="${8:-ubuntu}"
+UPDATED_BRANCH="${5:-master}"
+export CAPI_VERSION="${6:-v1alpha3}"
+export IMAGE_OS="${7:-Ubuntu}"
+export DEFAULT_HOSTS_MEMORY="${8:-4096}"
+DISTRIBUTION="${9:-ubuntu}"
+
+# Since we take care of the repo tested here (to merge the PR), do not update
+# the repo in metal3-dev-env 03_launch_mgmt_cluster.sh
+export FORCE_REPO_UPDATE=false
+
+# Clone the source repository
+cd "/home/${USER}"
+git clone ${UPDATED_REPO} tested_repo
+cd tested_repo
+git fetch origin
+git remote add upstream "https://github.com/${REPO_ORG}/${REPO_NAME}.git"
```

Comment:
Isn't it the upstream is going to be the same as the origin (UPDATED_REPO) in this case?

Reply:
no, UPDATED_REPO points to the source repo of the PR, for example Nordix, or whatever fork. While the upstream points to the target repo : for example metal3-io metal3-dev-env